### PR TITLE
Node.js Week2 Homework added(EMRE)

### PR DESCRIPTION
**Pull request by arasmr**
I think it is OK.

### CODE REVIEW

File: week2/homework/src/index.js [hunk 1]
```diff
@@ -1,3 +1,129 @@
...
+            } else {
+                resolve(data);
+            };
+        })
+    );
+}
+
+function listToDo() {
+    readFile()  
+        .then(data => { data !== '' ? console.log(`\nTo - Dos: \n\n${data}`) : console.log('\nThere is nothing in the To-Do list!\n') })
```
Comment (GeorgeSapkin) [inline week2/homework/src/index.js]: Try to break-up longer lines into multiple lines for better readability.

File: week2/homework/src/index.js [hunk 2]
```diff
@@ -1,3 +1,129 @@
...
+            };
+        })
+        .catch(console.error);
+}
+
+function removeToDo() {
+    readFile()
+        .then(data => {
+            let myData = data.split('\n');
+            myData.splice(input - 1, 1);
```
Comment (GeorgeSapkin) [inline week2/homework/src/index.js]: What if I call `remove -100`? Also it's better to pass values as function arguments and not as global variables like `input`.

File: week2/homework/src/index.js [hunk 3]
```diff
@@ -1,3 +1,129 @@
...
+
+function printHelp() {
+    console.log(`
+    HackYourFuture Node.js Week 2 - Lecture To-Do App
+
+    Options:
+
+    add    [to-do]                   add a to-do item
+    help                             show help text
+    remove [line index] + ['to-do']  remove to-do
```
Comment (GeorgeSapkin) [inline week2/homework/src/index.js]: You probably meant `remove [line index]   remove to-do`.

File: week2/homework/src/index.js [hunk 4]
```diff
@@ -1,3 +1,129 @@
...
+}
+
+function updateToDo() {
+    readFile()
+        .then((data) => {
+            let willUpdate = data.split('\n');
+            if (isNaN(input)) {
+                printHelp();
+            } else {
+                console.log(`\nDone!  "${willUpdate[input - 1]}" is updated as "${update}"\n`);
```
Comment (GeorgeSapkin) [inline week2/homework/src/index.js]: What if I call `update -100 something`? Also it's better to pass values as function arguments and not as global variables like `input` and `update`.

File: week2/homework/src/index.js (hunk 3 above)
Comment (arasmr) [reply week2/homework/src/index.js]: Yeah. I meant remove(line index).

**Pull request closed without merging**